esphome/components/ethernet/ethernet_component.cpp

```diff
@@ -341,8 +341,10 @@ void EthernetComponent::dump_config() {
   ESP_LOGCONFIG(TAG, "  Reset Pin: %d", this->reset_pin_);
   ESP_LOGCONFIG(TAG, "  Clock Speed: %d MHz", this->clock_speed_ / 1000000);
 #else
-  if (this->power_pin_ != -1) {
-    ESP_LOGCONFIG(TAG, "  Power Pin: %u", this->power_pin_);
+  {
+    if (this->power_pin_ != -1) {
+      ESP_LOGCONFIG(TAG, "  Power Pin: %u", this->power_pin_);
+    }
```

Comment:
Not really necessary here
```suggestion
  if (this->power_pin_ != -1) {
    ESP_LOGCONFIG(TAG, "  Power Pin: %u", this->power_pin_);
```